velox/type/Timestamp.h

```diff
@@ -33,9 +33,26 @@ namespace facebook::velox {
 struct Timestamp {
  public:
   enum class Precision : int { kMilliseconds = 3, kNanoseconds = 9 };
+  // We need to limit the range of seconds to avoid some problems.
+  // Seconds in Timestamp need to be in range [INT64_MIN/1000, INT64_MAX/1000].
+  // Presto's Timestamp is stored in milliseconds, this range ensures that
+  // Timestamp's range in Velox will not be smaller than Presto, and can make
+  // Timestamp::toString work correctly.
+  static constexpr int64_t kMaxSeconds =
```

Comment:
Would you add a test to ensure that toString() succeeds for Timestamp(kMaxSeconds, kMaxNanos) and Timestamp(kMinSeconds, 0)?